### PR TITLE
Prototype FlatSat Testing

### PR DESCRIPTION
**Pull request by tganley**

### CODE REVIEW

File: equipment_handler/include/housekeeping_athena.h [hunk 1]
```diff
@@ -38,20 +39,21 @@
...
+  uint16_t  OBC_uptime;             //Seconds=value*10 , Max=655360 sec (7.6 days)
+  uint8_t   solar_panel_supply_curr;
+  uint8_t   OBC_software_ver;
+  uint16_t  cmds_received;          //Total # since last boot
+  uint16_t  pckts_uncovered_by_FEC; //Total # since last boot
 } athena_housekeeping;
 
-int Athena_getHK(athena_housekeeping *athena_hk);
-int Athena_hk_convert_endianness(athena_housekeeping *athena_hk);
+int Athena_getHK(athena_housekeeping* athena_hk);
```
Comment (Taylorr82) [inline equipment_handler/include/housekeeping_athena.h]: Was clang run on this file? the * should be on the parameter, not the type

File: equipment_handler/source/housekeeping_athena.c [hunk 2]
```diff
@@ -41,17 +43,23 @@ int HAL_get_temp_all(long *temparray) {
...
  */
 uint16_t Athena_get_OBC_uptime() {
-    TickType_t OBC_ticks = xTaskGetTickCount(); // 1 tick/ms
-    uint32_t OBC_uptime_s = (OBC_ticks * portTICK_PERIOD_MS) / 1000;
 
-    // convert OBC_uptime from 32 bit to 16 bit. Max = 655350 seconds (7.6 days)
-    // Unit will be deca-seconds due to size restraint, seconds = OBC_uptime*10
-    return (uint16_t)OBC_uptime_s / 10;
+  TickType_t OBC_ticks = xTaskGetTickCount(); //1 tick/ms
+  uint32_t OBC_uptime_32 = OBC_ticks/1000;
```
Comment (Taylorr82) [inline equipment_handler/source/housekeeping_athena.c]: Should use the macro for this

File: equipment_handler/source/housekeeping_athena.c [hunk 3]
```diff
@@ -41,17 +43,23 @@ int HAL_get_temp_all(long *temparray) {
...
+  if (OBC_uptime_32 > 655350) {
```
Comment (Taylorr82) [inline equipment_handler/source/housekeeping_athena.c]: Will this ever return true?

File: equipment_handler/source/spi_io.c [hunk 4]
```diff
@@ -31,29 +29,30 @@ void SPI_Release(void) {
...
 inline void SPI_CS_Low(uint8_t bVolNum) {
     if (bVolNum == 0) {
-#ifdef IS_ATHENA
-        gioSetBit(gioPORTA, 3, 0); // CS LOW
-        gioSetBit(hetPORT2, 6, 1); // CS HIGH
-#else
-        gioSetBit(hetPORT1, 12, 0); // CS LOW
-        gioSetBit(hetPORT1, 14, 1); // CS HIGH
+    #ifdef IS_ATHENA
+        //gioSetBit(gioPORTA, 3, 0); //CS LOW
```
Comment (Taylorr82) [inline equipment_handler/source/spi_io.c]: Why is this commented out?

File: equipment_handler/source/tempsense_athena.c [hunk 5]
```diff
@@ -37,6 +37,7 @@ int gettemp_all(long *temparray) {
     int i;
     for (i = 0; i < NUM_TEMP_SENSOR; i++) {
         tmp421_read(tmp_addr[i], CHANNEL_LOCAL, &temparray[i]); // assuming we want to read remote channel
+        for (delay = 0; delay < 0x1000; delay++);//temporary fix... don't want delay down the road
```
Comment (Taylorr82) [inline equipment_handler/source/tempsense_athena.c]: This should use vTaskDelay()

**Review by Taylorr82 — CHANGES_REQUESTED** [review]

File: equipment_handler/source/spi_io.c (hunk 4 above)
Comment (tganley) [reply equipment_handler/source/spi_io.c]: This is what we did when we were trying to get the SD card to work. @joshdellaz should we keep this?

File: equipment_handler/source/housekeeping_athena.c (hunk 3 above)
Comment (tganley) [reply equipment_handler/source/housekeeping_athena.c]: I don't see why it couldn't. But this seems like a weird way to do things. @nsorense4 do you know why it's written like this?

File: equipment_handler/source/housekeeping_athena.c (hunk 2 above)
Comment (tganley) [reply equipment_handler/source/housekeeping_athena.c]: Done

File: equipment_handler/include/housekeeping_athena.h (hunk 1 above)
Comment (tganley) [reply equipment_handler/include/housekeeping_athena.h]: I'm really struggling to figure out how to get clang to work on Windows

Comment (tganley) [reply equipment_handler/include/housekeeping_athena.h]: Nevermind, got it.

File: equipment_handler/source/spi_io.c (hunk 4 above)
Comment (Taylorr82) [reply equipment_handler/source/spi_io.c]: This is supposed to set the CS for both pins because we have two SD cards

Comment (tganley) [reply equipment_handler/source/spi_io.c]: I will uncomment this and then make sure the file system is still working today.

**Review by Taylorr82 — APPROVED** [review]

**Pull request merged**